packages/contracts-sdk/src/lib/contracts-sdk.ts

```diff
@@ -422,164 +436,185 @@ export class LitContracts {
-        this.signer
-      ) as unknown as multisenderContract.Multisender &
-        multisenderContract.Multisender,
-    };
+    if (addresses.Allowlist.abi) {
+      this.allowlistContract = {
+        read: new ethers.Contract(
+          addresses.Allowlist.address,
+          addresses.Allowlist.abi as any,
+          this.provider
+        ) as unknown as allowlistContract.Allowlist &
+          allowlistContract.Allowlist,
+        write: new ethers.Contract(
+          addresses.Allowlist.address,
+          addresses.Allowlist.abi as any,
+          this.signer
+        ) as unknown as allowlistContract.Allowlist &
+          allowlistContract.Allowlist,
+      };
+    }
 
-    this.pkpHelperContract = {
-      read: new ethers.Contract(
-        addresses.PKPHelper.address,
-        addresses.PKPHelper.abi as any,
-        this.provider
-      ) as unknown as pkpHelperContract.PKPHelper & pkpHelperContract.PKPHelper,
-      write: new ethers.Contract(
-        addresses.PKPHelper.address,
-        addresses.PKPHelper.abi as any,
-        this.signer
-      ) as unknown as pkpHelperContract.PKPHelper & pkpHelperContract.PKPHelper,
-    };
+    if (addresses.LITToken.abi) {
+      this.litTokenContract = {
+        read: new ethers.Contract(
+          addresses.LITToken.address,
+          addresses.LITToken.abi as ethers.ContractInterface,
+          this.provider
+        ) as unknown as litTokenContract.LITToken & litTokenContract.LITToken,
+        write: new ethers.Contract(
+          addresses.LITToken.address,
+          addresses.LITToken.abi as ethers.ContractInterface,
+          this.signer
+        ) as unknown as litTokenContract.LITToken & litTokenContract.LITToken,
+      };
+    }
 
-    this.pkpNftContract = {
-      read: new ethers.Contract(
-        addresses.PKPNFT.address,
-        addresses.PKPNFT.abi as any,
-        this.provider
-      ) as unknown as pkpNftContract.PKPNFT & pkpNftContract.PKPNFT,
-      write: new ethers.Contract(
-        addresses.PKPNFT.address,
-        addresses.PKPNFT.abi as any,
-        this.signer
-      ) as unknown as pkpNftContract.PKPNFT & pkpNftContract.PKPNFT,
-    };
+    if (addresses.Multisender.abi) {
+      this.multisenderContract = {
+        read: new ethers.Contract(
+          addresses.Multisender.address,
+          addresses.Multisender.abi as ethers.ContractInterface,
+          this.provider
+        ) as unknown as multisenderContract.Multisender &
+          multisenderContract.Multisender,
+        write: new ethers.Contract(
+          addresses.Multisender.address,
+          addresses.Multisender.abi as ethers.ContractInterface,
+          this.signer
+        ) as unknown as multisenderContract.Multisender &
+          multisenderContract.Multisender,
+      };
+    }
+    if (addresses.PKPHelper.abi) {
+      this.pkpHelperContract = {
+        read: new ethers.Contract(
+          addresses.PKPHelper.address,
+          addresses.PKPHelper.abi as any,
```

Comment:
```suggestion
          addresses.PKPHelper.abi as ethers.ContractInterface,
```
Same with the other `any`s in `Contract` constructor